backend/backendserver/src/main/java/com/github/klucsik/birdnoiseserver/backendserver/services/DevicePlayParamService.java

```diff
@@ -27,7 +29,6 @@
     public DevicePlayParam save(DevicePlayParam devicePlayParam) throws MethodArgumentNotValidException {
 
         devicePlayParam.setDevice(deviceService.GetById(devicePlayParam.getDevice().getId()));
-
         devicePlayParam.setPlayParam(playParamService.getOne(devicePlayParam.getPlayParam().getId()));
 
```

Comment:
When I create a new DPP, should it became draft by default? Now it'll become null when I save something on API

Reply:
I see. When I sent "status":"DRAFT" in postman it became null. Did I make a typo or that part is not working?

Reply:
If you type: "status": "DRAFT" then it should be all right also you need to give everything else because it will run into nullpointer exception.